### ISSUE TITLE
用的第三方包出错了，有些方法已经被弃用了。

### ISSUE THREAD

**Issue by skiy** [go-xiaohei/pugo#30]
这方法已经弃用了。。。
```
pugo build
DEPRECATED cli.App.RunAndExitOnError.  This is an error in the application.  Please contact the distributor of this application if this is not you.  See https://github.com/urfave/cli/blob/master/CHANGELOG.md#deprecated-cli-app-runandexitonerror
INFO|08-01 00:19:19.8126|Build|Start
INFO|08-01 00:19:19.8127|Build|Source|source
INFO|08-01 00:19:19.8268|Build|Theme|theme/default
INFO|08-01 00:19:19.9417|Build|8 Pages
INFO|08-01 00:19:19.9418|Build|Done|1|129.3ms
DEPRECATED Action signature.  Must be `cli.ActionFunc`.  This is an error in the application.  Please contact the distributor of this application if this is not you.  See https://github.com/urfave/cli/blob/master/CHANGELOG.md#deprecated-cli-app-action-signature
```

**Comment by fuxiaohei** [comment]
I had updated it in `develop` branch.
It can be resolved in next release version.

**Comment by fuxiaohei** [comment]
I had released `0.10.5`, fixed.
